apiserver/facades/client/charmhub/convert.go

```diff
@@ -195,12 +199,12 @@ func transformRelations(requires, provides map[string]charm.Relation) map[string
 	return relations
 }
 
-func formatRelationPart(rels map[string]charm.Relation) (map[string]string, bool) {
-	if len(rels) <= 0 {
+func formatRelationPart(r map[string]charm.Relation) (map[string]string, bool) {
+	if len(r) <= 0 {
```

Comment:
```suggestion
	if len(r) == 0 {
```

Can't really be <0 ;-)

Reply:
Smiley face with a hat? /s

Reply:
🧙‍♂️